pages/three.js

```diff
@@ -30,9 +30,8 @@ var ThreePreview = function() {
 
         scene = new THREE.Scene();
 
-        defaultCamera = new THREE.PerspectiveCamera(45, container.offsetWidth / container.offsetHeight, 1, 20000);
+        defaultCamera = new THREE.PerspectiveCamera(45, container.offsetWidth / container.offsetHeight, 1e-5, 1e10);
```

Comment:
I noticed that about Boombox, but I think that's something that changed with the Boombox model itself.  The near/far planes here don't affect the default camera distance, and some other models seem to have reasonable distances.

The numbers here aren't particularly magical, they're just "too small to matter" and "too large to matter" respectively.  The magic is below, where I added `logarithmicDepthBuffer: true`, this allows otherwise unreasonably large distances between these numbers.